docs/api/covidcast.md

```diff
@@ -153,6 +153,15 @@ You should specify only one of these three parameters in any given query.
 remain the same as a previous issue. If you have a research problem that would
 require knowing when we last confirmed an unchanged value, please get in touch.
 
+#### Useful Syntax Features
+
+The following params help specify multiple source-signal, timetype-timevalue or geotype-geovalue pairs. Use them in replacement of the usual required params.
+
+| Parameter | Format | Description | Example |
```

Comment:
this is a good layout!  it might be easier to understand what its good for if you put it right underneath the "Required" table and named it something related to "alternatives", plus add a column called "Replacement For"